apps/portals/b2ai.standards/src/assets/d4d/B2AI_ORG_114_d4d.css

```diff
@@ -0,0 +1,170 @@
+body {
```

Comment:
all 3 CSS files are identical, so if we are using them, we probably only need one.

Reply:
Good catch! I asked on slack if we need to have separate ones. I doubt it. I'll need to change (simplify) some code to make them all use a single css. Do you want me to do that before you merge, or do it as a small, separate PR?